cfpb/teachers-digital-platform
# Fixed Modal dialogs interfering with each other

The two modal dialogs we have were interfering with each other because they were accidentally using each others functions.

## Additions

- Duplicated the JavaScript functions that handles the click outside for the Modal Dialogs
- This was performed twice
  - On the Start page in pure JavaScript
  - On the Survey pages (in the dialogs/ folder)

## Testing

- Load the start page and verify
  - [ ] You can show both of the modal dialogs and close using the normal close buttons
  - [ ] You can close them by clicking outside of them (all 4 sides)
- Load any of the survey pages
  - [ ] You can show both of the modal dialogs and close using the normal close buttons
  - [ ] You can close them by clicking outside of them (all 4 sides)
  

## Review

- @dcmouyard 

[Preview this PR without the whitespace changes](?w=0)


## Checklist
